pkg/kube/util.go

```diff
@@ -302,3 +304,30 @@ func StripUnusedFields(obj any) (any, error) {
 	t.GetObjectMeta().SetManagedFields(nil)
 	return obj, nil
 }
+
+func SlowConvertKindsToRuntimeObjects(in []crd.IstioKind) ([]runtime.Object, error) {
```

Comment:
Is this the right place?  I think it should be along with RuntimeObjectToKind convert

Reply:
had to move otherwise it adds 20mb to pilot-agent binary

Reply:
How did you find this increase

Reply:
We have a test for it